packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -62,17 +67,11 @@ class PinCodeBloc extends $PinCodeBloc {
   final BehaviorSubject<String> _pinCode = BehaviorSubject.seeded('');
 
   @override
-  Stream<int> _mapToDigitsCountState() => Rx.merge([
-        _$addDigitEvent.switchMap((digit) => _addDigit(digit).asResultStream()),
-        _$deleteDigitEvent.switchMap(
-          (_) {
-            _pinCode.add(_pinCode.value.substring(
-                0, _pinCode.value.isNotEmpty ? _pinCode.value.length - 1 : 0));
-            return Stream.value(_pinCode.value.length);
-          },
-        ).asResultStream(),
-        errorState.mapTo(0).asResultStream(),
-      ]).whereSuccess().startWith(0).share();
+  Stream<int> _mapToDigitsCountState() => _pinCode
+      .map<int>((pinCode) => pinCode.length)
+      .asResultStream()
```

Comment:
Usually the result stream is used on asynchronous calls. From what I see this is redundant

Reply:
Removed ResultStream